tests/bcl-test/BCLTests/Info.plist

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleName</key>
+	<string>BCLTests</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.xamarin.BCLTests</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleVersion</key>
+	<string>1.0</string>
+	<key>LSRequiresIPhoneOS</key>
+	<true/>
+	<key>MinimumOSVersion</key>
+	<string>8.0</string>
```

Comment:
Also, do generate a plist per project so that we do not have several BCLTests apps but one app per test solution, so
SystemXmlAndSystemLINQXml would be an example (testing two diff assemblies).